crnk-documentation/src/docs/asciidoc/operations.adoc

```diff
@@ -40,6 +40,30 @@ There is further an operations client implementation that works along the regula
 include::../../../../crnk-operations/src/test/java/io/crnk/operations/OperationsPostTest.java[tags=client]
 ----
 
+## Operation Module Properties
+
+The operations module can be customized using one of the following properties:
+
+.Operation Module Properties
+|===
+|*Property Name* |*Description* |*Default Value*
+|resumeOnError | If an error occurs during the operation execution, continue the other operations. | False
+|includeAllRelationships | Toggle to include all relationships in the request by default. | True
```

Comment:
naming of this property should change I think. It is more a "includeChangedRelationships". So everything that gets POSTED/PATCHED will be returned as such.

Looking at it, it would be good to update the implementation of that feature (independent of this PR). Currently it triggers a full inclusion, it rather should trigger SerializeType.ONLY_ID.

Reply:
Hi @remmeier,

I'm also thinking more about this now, and I'm confused on the purpose of the include parameter on this line:

`parameters.put("include", getLoadedRelationshipNames(resource));`

When actually running the operation with a bunch of POST requests, it will return a response with "self" and "related" links for each of the relationships. And the response does not have a "included" section to it. So it seems in my case, it's running the SQL for the include, but not actually displaying the include in the response back.

It seems that the fetchUpToDateResponses method is performing the include but not displaying the results of the include. 

Perhaps the "include" parameter should just be removed all together? Or I might be misunderstanding the purpose of it.

Thanks in advance!

Reply:
it currently triggers an "inclusion" to get hands on the IDs and throwing everything else away. So the results will have relationship/data-elements holding the IDs of POSTed/PATCHed resources.

I created https://github.com/crnk-project/crnk-framework/issues/559 to address it. It should not do any relationship loading, but if possible still display the relationship IDs even if it is not EAGER/ID_ONLY.

Reply:
Ah I see, that makes sense. Okay, I will change the name of the toggle as suggested above and it should be ready soon.

Thank you for explaining that